PWGCF/Tasks/CMakeLists.txt

```diff
@@ -14,6 +14,11 @@ o2physics_add_dpl_workflow(dptdptcorrelations
                     PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore O2Physics::PWGCFCore
                     COMPONENT_NAME Analysis)
 
+o2physics_add_dpl_workflow(multiparticle-correlations-ab
+                    SOURCES multiparticle-correlations-ab.cxx
+                    PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore O2Physics::PWGCFCore
+                    COMPONENT_NAME Analysis)
+
```

Comment:
It is already there: https://github.com/AliceO2Group/O2Physics/pull/984/commits/3341854c3ab9426335e334a0505a300e12a531f9#diff-3709836f9f82a85378f76f6eb85e842c5d7ab4b765ae06523562205adce89d20

Since at the moment I only compile the task directly, the other `CMakeLists.txt` files are there, but only with the dummy content (copyrights). They will be populated later, when more source code is added.